### ISSUE TITLE
Consider Native WebRTC Implementation

### ISSUE THREAD

**Issue by willscott** [keroserene/go-webrtc#26]
Potentially a huge amount of work, but it would be great to see an RTC implementation that was more than an adapter to the c-native reference implementation. Reasons include gaining Go's safety and compilation benefits, independent failure / exploitation surface area, and could likely be significantly easier to understand.

**Comment by keroserene** [comment]
Agreed for those potential benefits (and having another true independent implementation in the ecosystem would be good for the spec)... though it would indeed require an enormous effort.

Right now, having made WebRTC work in Go at all has been wonderful for getting other things off the ground :) But, perhaps in the future!

**Comment by backkem** [comment]
For your information: We built a pure Go WebRTC implementation at [pions/webrtc](https://github.com/pions/webrtc).
